hadoop-ozone/dist/src/main/compose/ozonesecure-ha/docker-compose.yaml

```diff
@@ -49,7 +49,7 @@ services:
       - ./krb5.conf:/etc/krb5.conf
       - ../../libexec/transformation.py:/opt/transformation.py
     environment:
-      HADOOP_CONF_DIR: /opt/hadoop/etc/hadoop
+      OZONE_CONF_DIR: /opt/hadoop/etc/hadoop
```

Comment:
This service runs `hadoop kms`, which is Hadoop, not Ozone.  This occurrence should not be changed.